lib/rules/best-practises/no-empty-blocks.js

```diff
@@ -50,18 +89,47 @@ class NoEmptyBlocksChecker extends BaseChecker {
   }
 
   Block(node) {
-    // ignore empty constructor body when it's being defined to call a parent
-    // constructor
+    // ignore empty constructor body when it's calling a parent
     if (node.parent.isConstructor && node.parent.modifiers.length > 0) {
       return
     }
 
-    const isFallbackFunctionBlock = isFallbackFunction(node.parent)
-    const isReceiveFunctionBlock = isReceiveFunction(node.parent)
-    if (isFallbackFunctionBlock || isReceiveFunctionBlock) {
-      // ignore empty blocks in fallback or receive functions
+    // empty try block
+    if (node.parent?.type === 'TryStatement') {
+      if (node.parent.body === node && this.allowEmptyTry) return
```

Comment:
in which case would `node.parent.body !== node` ? could the block be another attribute? in that case, would it make more sense to assert on the type of the parent?

Also if I remove that comparison all tests continue to pass